package.json

```diff
@@ -9,8 +9,11 @@
     "build": "next build",
     "start": "next start -p $PORT",
     "build-static": "next build && next export",
-    "test": "jest --coverage",
+    "cy:open": "cypress open",
+    "test:unit": "jest --coverage __tests__/unit",
```

Comment:
How are we going to find integration tests coverage?

Reply:
We don't have integration testing yet. we will do e2e testing